src/libs/storage/widgetSessionCache.ts

```diff
@@ -0,0 +1,55 @@
+import { localStorageHelper } from '../../utils';
+
+// const DEFAULT_VALUE = { userId: null, channelUrl: null, expireAt: 0 };
+const WIDGET_SESSION_PREFIX = '@sendbird/chat-ai-widget';
+const getKey = (appId: string, botId: string) => {
+  return `${WIDGET_SESSION_PREFIX}/${appId}/${botId}`;
+};
+
+export type WidgetSessionCache = {
+  userId: string;
+  channelUrl: string;
+  expireAt: number;
+  sessionToken?: string;
+  strategy?: 'auto' | 'manual'; // default: auto
```

Comment:
```suggestion
  strategy: 'auto' | 'manual'; // default: auto
```
It seems there's no chance that `strategy` gets undefined.

Reply:
Since this is a newly added value, the cached values for existing customers will be `undefined`

Reply:
I've added default value

-> https://github.com/sendbird/chat-ai-widget/pull/238/commits/04b0a1b3f818ecaa9e1985384c07da1c431bb45d